Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN curl -sS -o - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-ke
 RUN echo "deb [arch=amd64]  http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list
 RUN apt-get -y update && apt-get -y install google-chrome-stable
 # Install Selenium
-RUN curl https://chromedriver.storage.googleapis.com/2.41/chromedriver_linux64.zip -o  /usr/bin/chromedriver_linux64.zip
+RUN curl https://chromedriver.storage.googleapis.com/81.0.4044.69/chromedriver_linux64.zip -o  /usr/bin/chromedriver_linux64.zip
```

Comment:
Why did we need to do this? I am just curious. Was it needed, and why was the old one `2.41` and the new one actually a chrome version?

Reply:
So in our Dockerfile, we run: apt-get install google-chrome-stable, whose latest version is 81 and our chromedriver (both binary and the npm module) is fixed at 79. So presubmit failed.
 
I don't know why we haven't seen this before. My best guess is that we use cached Docker images which contains the 79 version of chrome.

So either: we updated to support chrome 81 or we change to always use chrome 79 (probably need to change the apt-get command).